std_msgs/Header header

int32 xian_dj_retractable_platform_stand_linear_actuator_enble 
int32 xian_dj_retractable_platform_stand_linear_actuator_move 
int32 xian_dj_retractable_platform_first_linear_actuator_enble 
int32 xian_dj_retractable_platform_first_linear_actuator_move 
int32 xian_dj_retractable_platform_second_linear_actuator_enble 
int32 xian_dj_retractable_platform_second_linear_actuator_move 
